userContactInfo.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$id": "https://getdock.github.io/schemas/userContactInfo.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Contact information for the user.",
+  "title": "User Contact Information",
+  "allOf": [
+    {
+      "$ref": "https://getdock.github.io/schemas/dock.json"
+    }
+  ],
+  "type": "object",
+  "properties": {
+    "country": {
+      "description": "User's country",
+      "type": "string"
+    },
+    "city": {
+      "description": "User's city",
+      "type": "string"
+    },
+    "phone": {
+      "description": "User's phone number",
+      "type": "number"
```

Comment:
Don't know if this should be a number. Current representation eliminates all phones that have extensions.